deploy/docker/cp-api-srv/init-api

```diff
@@ -226,6 +226,12 @@ if [ -f "$win_code_sign_cert_path" ]; then
         "$CP_WIN_CODE_SIGN_CERT_DESC" "$CP_WIN_CODE_SIGN_CERT_URL" "$CP_API_SRV_STATIC_DIR"
 fi
 
+echo "Copying DTS jar file to static resources"
```

Comment:
I think it would be nice to fix `cp-api-srv` Dockerfile and remove all non-relevant jars, everything except for:
- pipeline.jar
- jwt-generator.jar
- data-transfer-service.jar